src/utils.c

```diff
@@ -1651,7 +1651,7 @@ const gchar *utils_get_default_dir_utf8(void)
  *  @param argv The child's argument vector.
  *  @param env The child's environment, or @c NULL to inherit parent's.
  *  @param flags Ignored.
- *  @param child_setup Ignored.
+ *  @param child_setup @girskip Ignored.
  *  @param user_data Ignored.
```

Comment:
shouldn't this be also ignored?  or is it automatically because it's the data of the previous func, even if it's explicitly ignored?